repair/featurize/freqfeat.py

```diff
@@ -25,7 +25,7 @@ def gen_feat_tensor(self, input, classes):
         for idx, val in enumerate(domain):
             try:
                 prob = float(self.single_stats[attribute][val])/float(self.total)
-            except:
+            except Exception:
```

Comment:
I assume this catches `ZeroDivisionError`. If so, add it explicitly

Reply:
This actually catches the case where a `KeyError` is thrown: when the `val` in the `domain` does not appear in the original dataset when calculating stats (I'm actually not sure how this could be the case currently since we always generate domain from the values in the corresponding attribute column, yet I'm running into this issue when running the tests. Do you have context on this @thodrek ?)

Reply:
Can you investigate if there is a problem with the groupby statement over the data frame that generates the counts that go into single_stats attributes? I have also observed the KeyError exception but never investigated properly. Grab a key for which you get an error and see what is happening in the group_by statement over the raw pandas that generates the single_stats data frame.

Reply:
Added a TODO and opened #26.